src/components/Header/Header.tsx

```diff
@@ -1,20 +1,72 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import cn from "classnames"
 import Container from "../Container/Container"
 import { ReactComponent as Logo } from "./assets/logo.svg"
+import NavigationLinks from "../NavigationLinks/NavigationLinks"
+import MenuMobile from "./MenuMobile"
+import LinkLaunchApp from "../Link/LinkLaunchApp"
 import s from "./Header.module.css"
 
 interface IHeaderProps {
   className?: string
 }
 
 const Header: React.FC<IHeaderProps> = ({ className }) => {
+  const [isMenuOpened, setIsMenuOpened] = useState(false)
+  useEffect(() => {
+    if (isMenuOpened) {
+      document.body.style.overflow = "hidden"
```

Comment:
It is best to avoid directly manipulating the DOM because it can conflict with React's updates